lib/duty_calculator/helper.rb

```diff
@@ -0,0 +1,20 @@
+module DutyCalculator
+  module Helper
+    class << self
+      def iterate(h)
+        h.each do |k,v|
+          value = v || k
+          if value.is_a?(Hash) || value.is_a?(Array)
+            iterate(value)
+          end
+          if k == '__content__' && v.to_i
```

Comment:
What is the purpose of the `to_i` check? I think it just is always 0 for non-floaty string, which is truthy.